internal/core/model.rs

```diff
@@ -968,7 +974,21 @@ impl<C: RepeatedComponent> Repeater<C> {
         self.inner.borrow().components.len()
     }
 
-    /// Return true if the Repeater is empty
+    /// Return the amount of item currently in the component
```

Comment:
i guess this comment is not accurate.

I was wondering why you used a range in the vtable instead of just the size,  until i saw the implementation.

Reply:
We could discuss what we want to do about that support for offset we have in some places, but that is outside of this patch I think:-)